src/crypto/isa-l/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+set(isal_dir ${CMAKE_SOURCE_DIR}/src/crypto/isa-l/isa-l_crypto)
```

Comment:
can use `CMAKE_CURRENT_SOURCE_DIR` to refer to the directory where this `CMakeLists.txt` lives, so
```cmake
set(isal_dir ${CMAKE_CURRENT_SOURCE_DIR}/isa-l_crypto)
```